### DISCUSSION TITLE
Easy(er) color selection

### DISCUSSION THREAD

**Discussion by luc0x61** [sumatrapdfreader/sumatrapdf#3498] [Ideas]
I would like to have an easier color selection for document **borders**, and maybe even the selection of text/background of the document itself.
Maybe a direct color selector for some peculiar items?

**Comment by GitHubRulesOK** [comment]
generally there are few documents that have a border definition so PDF background is technically a void with a define colourless shape which viewers can set to one canvas colour usually white so that all other colours can be seen in full contrast, it would make no sense to set canvas to black variations perhaps just a light static grey with a hue of "warm" ?

The spare space between media crop boundary and windows pane for PDF can be set using "GradientColors" to a set of one two or three of your choice but would not be much value if constantly flashing as changes.

**Comment by luc0x61** [comment]
My fault: what I meant is the window background around borders, the one set with the property `MainWindowBackground` 😄

**Comment by GitHubRulesOK** [comment]
Main Window Background should be Yellow like below, but users complained it was too "Happy" as Watchman Yellow, so the default has been exposed as an advanced setting and the default of MainWindowBackground = #80fff200 has been neutered to a light shade of grey but its just as easy to edit like any others (just confusing that yellow as shown is not yellow!)

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/324dfe3b-d797-4981-bb71-af8dcf6e1597)

**Reply by luc0x61** [reply]
Yes, that's _easy_ to change, but I found myself every time scrolling through the settings, on a newly installed application (as I don't remember `MainWindowBackground`). Plus sometimes you already have many files opened, and each of them adds some more text to the settings, with a long file to scroll and look for which is which... Here's why do exist user settings dialog **windows**, to make easy things _easier_! 😄

**Reply by GitHubRulesOK** [reply]
IF you have lines of favourite entries like say here I want to keep this custom set in a new install
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/a52268d1-7599-44ca-b374-46c92acfa1fb)

You simply use that as the settings file at start-up and SumatraPDF will add all the others as defaults, note if you look at settings it will not appear to have worked until you start using file open, tabs etc. (any valid file will do)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0b2286f1-d2aa-4735-9cfc-10bdedce4b6c)
